webapp/lib/controller.dart

```diff
@@ -193,9 +194,16 @@ Map<String, model.NeedsReplyData> getLatestDataForProjects(List<model.NeedsReply
   return latestProjectData;
 }
 
-bool isProjectStale(model.NeedsReplyData projectData) {
+bool isProjectStale<T>(T projectData) {
```

Comment:
Since it's no longer looking only at the projects only, consider renaming to `isDataStale`